packages/strapi-plugin-graphql/services/type-definitions.js

```diff
@@ -378,7 +378,7 @@ const buildCollectionType = model => {
     if (actionExists(resolverOpts)) {
       _.merge(localSchema, {
         query: {
-          [`${singularName}(id: ID!)`]: model.globalId,
+          [`${singularName}(id: ID!)`]: `${model.globalId}!`,
```

Comment:
I don't think it's valid to assume non-null here, a valid query may return null when asked for an ID that does not exist.

Agree with both the non-null constraints added to the plural query below.